cwbi-auth-http-client/src/main/resources/hec/army/usace/hec/cwbi/auth/http/client/trustmanagers/cumulusServer.pem

```diff
@@ -0,0 +1,31 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
is this for cumulus or cwbi-auth?

Reply:
cwbi-auth. Renamed to cwbitAuthServer.pem